__tests__/app.test.js

```diff
@@ -217,4 +210,42 @@ describe('/api/topics', () => {
       });
   
   }) 
-})
\ No newline at end of file
+  describe('PATCH /api/articles/:article_id', () => {
+    const updateArticleVote ={ inc_votes : 10}
+    it('returns updated article by article_id',()=>{  
+      return request(app)
+            .patch('/api/articles/1')
+            .send(updateArticleVote)
+            .expect(200)
+            .then(({ body: {article_id,title, topic,author,body, created_at, votes, article_img_url}}) => {
+              expect(votes).toBe(110)
+              expect(article_id).toBe(1)
+              expect(title).toBe('Living in the shadow of a great man')
+              expect(topic).toBe('mitch')
+              expect(author).toBe('butter_bridge')
+              expect(body).toBe('I find this existence challenging')
+              expect(created_at).toBe('2020-07-09T20:11:00.000Z')
+              expect(article_img_url).toBe('https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700')
+          })
+    })
+    it('returns 400 for invalid article_id',()=>{
+      return request(app)
+      .patch('/api/articles/lkj')
+      .send(updateArticleVote)
+      .expect(400)
+      .then(({body})=>{
+        expect(body.error).toBe('Invalid article ID format');    
+    })
+    })
+      it('Returns 404 when article format is valid but it is not in database ', () => {
```

Comment:
Good tests so far! It would also be beneficial to add a test for an incomplete body
E.g. `{inc_votes: "notanumber"}`